samples/matter/common/src/bridge/ble_bridged_device.h

```diff
@@ -46,8 +44,11 @@ class BLEBridgedDeviceProvider : public BridgedDeviceDataProvider {
 
 	BLEBridgedDevice &GetBLEBridgedDevice() { return mDevice; }
 	void SetConnectionObject(bt_conn *conn) { mDevice.mConn = conn; }
+	void RemoveConnectionObject() { mDevice.mConn = nullptr; }
+	bool IsInitialized() { return mDevice.mIsInitialized; }
```

Comment:
I generally agree with the concept of marking first connection using additional flag, however I think naming it initialization is quite confusing. Provider is initialized before we request creating the BLE connection. We also have `InitializeBridgedDevice` method, so I would expect device is initialized after it was called, but rest of the API suggest something opposite. Maybe we could just name precisely to inform about the purpose e.g. `mInitiallyConnected`, `ConfirmInitialConnection`. Adding some docs to do not leave any doubts about the usage could be useful too.